src/Sylius/Bundle/ApiBundle/Resources/config/services/serializers.xml

```diff
@@ -143,10 +143,10 @@
             <tag name="serializer.normalizer" priority="64" />
         </service>
 
-<!--        <service id="Sylius\Bundle\ApiBundle\Serializer\CustomerDenormalizer">-->
-<!--            <argument type="service" id="clock" />-->
-<!--            <tag name="serializer.normalizer" priority="64" />-->
-<!--        </service>-->
+        <service id="Sylius\Bundle\ApiBundle\Serializer\Denormalizer\CustomerDenormalizer">
```

Comment:
I can't find it on our API brainstorm outcome, but shouldn't we use dotted service IDs?
```suggestion
        <service id="sylius_api.denormalizer.customer" class="Sylius\Bundle\ApiBundle\Serializer\Denormalizer\CustomerDenormalizer">
```